bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java

```diff
@@ -786,35 +803,20 @@ private boolean isInitializable(Thing thing, @Nullable ThingType thingType) {
             return true;
         }
 
-        ConfigDescription description = resolve(configDescriptionURIFunction.apply(prototype), null);
-        if (description == null) {
-            logger.debug("Config description for '{}' is not resolvable, assuming '{}' is initializable",
+        URI configDescriptionURI = configDescriptionURIFunction.apply(prototype);
+        if (configDescriptionURI == null) {
+            logger.debug("Config description URI for '{}' not found, assuming '{}' is initializable",
                     prototype.getUID(), targetUID);
             return true;
         }
 
-        List<String> requiredParameters = getRequiredParameters(description);
-        Set<String> propertyKeys = configuration.getProperties().keySet();
-        if (logger.isDebugEnabled()) {
-            logger.debug("Configuration of '{}' needs {}, has {}.", targetUID, requiredParameters, propertyKeys);
+        try {
+            configDescriptionValidator.validate(configuration.getProperties(), configDescriptionURI);
+        } catch (ConfigValidationException e) {
+            return false;
```

Comment:
Should we add a trace log here to allow users to see validation messages in the log file? This might help to find the culprit for #2710.

Reply:
See https://github.com/openhab/openhab-core/pull/2711